scripts/build-bpf.sh

```diff
@@ -28,7 +28,7 @@ set -x
 cd "${C_DIR}"
 export V="${V:-1}"
 make clean 
-make  "${@:2}" 
+make
```

Comment:
are you sure that you can drop the extra arguments to `make` here?

Reply:
There was no argument being provided. I think this was just left there from an older version of the build-script.
I will double  check that with an echo on the older version and trying to build the docker image. (though am also sure this properly builds the C)